welcome/index.adoc

```diff
@@ -84,32 +84,37 @@ Explore the following {product-title} installation tasks:
+- **xref:../installing/installing_on_prem_assisted/installing-on-prem-assisted.html[Install a cluster by using the Assisted Installer]**: The Assisted Installer is an installation solution that is provided on the Red Hat Hybrid Cloud Console. The Assisted Installer supports installing an {product-title} cluster on many platforms, but with a focus on bare metal, Nutanix, and vSphere infrastructures.
 
-ifndef::openshift-origin[]
-- **xref:../installing/installing_ibm_cloud_public/preparing-to-install-on-ibm-cloud.adoc#preparing-to-install-on-ibm-cloud[Install a cluster on {ibm-cloud-name}]**: You can install {product-title} on {ibm-cloud-name} on installer-provisioned infrastructure.
+- **xref:../installing/installing_with_agent_based_installer/installing-with-agent-based-installer.html#installing-ocp-agent_installing-with-agent-based-installer[Install a cluster by using the Agent-based Installer]**: You can use the Agent-based Installer to generate a bootable ISO image that contains the Assisted discovery agent, the Assisted Service, and all the other information required to deploy an {product-title} cluster. The Agent-based Installer leverages the advantages of the Assisted Installer in a disconnected environment.
 
-- **xref:../installing/installing_ibm_powervs/preparing-to-install-on-ibm-power-vs.adoc#preparing-to-install-on-ibm-power-vs[Install a cluster on {ibm-power-name} Virtual Server]**: You can install {product-title} on {ibm-power-name} Virtual Server on installer-provisioned infrastructure.
+- **xref:../installing/installing_on_prem_assisted/installing-on-prem-assisted.html#using-the-assisted-installer_installing-on-prem-assisted[Installing {product-title} with the Assisted Installer]**: The Assisted Installer is an installation solution that is provided on the Red Hat {hybrid-console}. The Assisted Installer supports installing an {product-title} cluster on many platforms, but with a focus on bare metal, Nutanix, and vSphere infrastructures.
 
-- **xref:../installing/installing_ibm_power/installing-ibm-power.adoc#installing-ibm-power[Install a cluster on {ibm-power-name}]**: You can install {product-title} on {ibm-power-name} on user-provisioned infrastructure.
-endif::openshift-origin[]
+- **xref:../installing/installing_with_agent_based_installer/installing-with-agent-based-installer.html#installing-ocp-agent_installing-with-agent-based-installer[Installing {product-title} with the Agent-based Installer]**: You can use the Agent-based Installer to generate a bootable ISO image that contains the Assisted discovery agent, the Assisted Service, and all the other information required to deploy an {product-title} cluster. The Agent-based Installer leverages the advantages of the Assisted Installer in a disconnected environment
```

Comment:
These two seem to be duplicates of the two above, with slightly different titles:
![image](https://github.com/openshift/openshift-docs/assets/61474374/b6e1ff33-209c-4965-b891-daba6913b67b)

Reply:
Removed duplicates.